app/src/main/res/raw/whitebalance_shader.glsl

```diff
@@ -0,0 +1,15 @@
+#extension GL_OES_EGL_image_external : require
+precision mediump float;
+
+uniform vec2 resolution;
+uniform sampler2D framebuffer1;
```

Comment:
Rename to better explain what is what in terms of actual picture and whitebalancing